utilities.tsx

```diff
@@ -0,0 +1,16 @@
+export const getHostname = () => {
+  const env = process.env.NODE_ENV
+  let hostname = ''
+  if (env == 'development') {
+    hostname = '127.0.0.1:3000'
+  } else if (env == 'production') {
+    hostname = ''  // TODO: get production hostname when available
+  }
+  return hostname
+}
+
+export const stringToSlug = (text: String) => {
```

Comment:
Can we talk about what this is for next time we get together? I see that it's used in the single recipe pages

Reply:
To insert a human readable descriptive string to URL paths. 
It converts the recipe name to a URI compatible string. The regex removes spaces and then removes symbols